framework/SupervisedLearning/ROMCollection.py

```diff
@@ -637,7 +649,12 @@ def __init__(self, messageHandler, **kwargs):
       self.raiseAMessage('No evalMode specified for clustered ROM, so defaulting to "truncated".')
       self._evaluationMode = 'truncated'
     self.raiseADebug('Clustered ROM evaluation mode set to "{}"'.format(self._evaluationMode))
-    # choice?
+
+    # add expected meta keys
+    #if self._evaluationMode == 'clustered':
+    #  self.addMetaKeys(['cluster_multiplicity'], {'cluster_multiplicity': [self._clusterVariableID]})
```

Comment:
remove this (we can keep the one below as a "reminder")

Reply:
cleared